oxidation/libparsec/crates/protocol/tests/test_vlob.rs

```diff
@@ -37,7 +37,37 @@ use libparsec_types::{Maybe, ReencryptionBatchEntry};
         )
     )
 )]
-#[case::with_sequester_blob(
+#[case::without(
+    (
+        // Generated from Python implementation (Parsec v2.12.1+dev)
```

Comment:
`Generated from Python` means generated with the old python-based schemas, is this really the case in 2.12.1 ?

Reply:
True

Reply:
Any reason why we don't genrate this vlob_create from the old Python schema ?
As long as we can do it, I'd rather do it to be extra careful (remember: the rust schema are not in production and they are precisely the one we should be worried about !)

Of course if this is a new test case added because there is a new field in the schema that is not supported by the Python schema it is fine to stick with the Rust schema (as long as we also already have a testcase with the old python schema ^^).

Reply:
You shouldn't be worried about a `None` case, I can generate from a Python implementation, but only the comment will differ and need more time (need to checkout an old branch) :thinking:

Reply:
![image](https://user-images.githubusercontent.com/36041853/192757236-48cd32ce-f44e-4ba1-b237-310cac0d8e62.png)
![image](https://user-images.githubusercontent.com/36041853/192758225-8028f5cd-466f-46d5-bd85-59fc03855c66.png)